widgets/mini_player/public/style.css

```
@@ -0,0 +1,225 @@
+.player-info {
+    display: flex;
+    min-width: 0;
+    flex-flow: column;
+}
+
+.player-track {
+    color: var(--homey-color-mono-1000);
+    font-size: var(--homey-font-size-default);
+    font-weight: var(--homey-font-weight-medium);
+    line-height: var(--homey-line-height-default);
+    /*overflow: hidden;*/
```

Stylelint (17.4.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)